admin/client.go

```diff
@@ -218,13 +217,22 @@ func (c *Client) ListChangeSummaries(
 		return nil, err
 	}
 
-	newDoc, err := document.NewInternalDocumentFromSnapshot(key, seq, snapshotMeta.Lamport, snapshotMeta.Snapshot)
+	newDoc, err := document.NewInternalDocumentFromSnapshot(
```

Comment:
In the `history_test_with_purging_changes` test, intermittently the sequence of newDoc is zero. If you look at the snapshotMeta information, the lamport and seq information are set to the initial values.

Reply:
It seems that I have found a suspicious situation.

At the server-side implementation, when the yorkie server is synced, the creation or reflection of changes is synchronous, but the snapshot creation seems to be a goroutine. 
If so, it seems that a history request is made before the snapshot creation is completed, and the snapshotMeta may be created based on ServerSeq 0.
In that case, newDoc will be created from 0 since there is no snapshot yet.

Reply:
I did the same analysis. You used `time.Sleep` in the current commit, but I don't think this completely solves the cause of the test failure.

How about testing using monkey-patch? 
example:
```go
	t.Run("history test with purging changes", func(t *testing.T) {
	        ...
		var b *background.Background
		// "bou.ke/monkey"
		monkey.PatchInstanceMethod(
			reflect.TypeOf(b),
			"AttachGoroutine",
			func(_ *background.Background, f func(c context.Context)) {
				f(context.Background())
			},
		)
		...
	}
```

But the problem is that bou.ke/monkey has an issue in an ARM machine.
reference: https://github.com/jenkins-x/jx/issues/2081

In my case, I checked the improvement by running the test with the command below referring to the reference above.

`GOARCH=amd64 go test -tags integration -gcflags=all=-l ./test/integration/main_test.go ./test/integration/history_test.go`

cc. @hackerwins

Reply:
Thanks for the nice comment .
It seems like a very good way to explicitly handle it!!

https://github.com/yorkie-team/yorkie/blob/4b333b949e03ae3f60fdbeb406a45c0c1a415541/test/integration/snapshot_test.go#L59-L63

However, other tests also seem to use `time.Sleep`. It seems good to apply monkey-patch consistently for these tests as well. 
Would it be better to treat this as a seperate issue?

Reply:
It would be good to deal with other issues.

Reply:
It would be good to consider the following:

1. Call `unpatch` explicitly by returning guard.: https://github.com/bouk/monkey#using-monkey
2. Add `amd64` tag to run only on `amd64` machine: https://github.com/yorkie-team/yorkie/issues/388

Reply:
Thank you for comment!
I updated the code.
Can you check if the tag is working properly in the ARM environment?